lib/pos_epoch_complete_hook.go

```diff
@@ -140,12 +140,28 @@ func (bav *UtxoView) runEpochCompleteSnapshotGeneration(epochNumber uint64) erro
 	return nil
 }
 
+// simulateNextEpochEntry simulates the block range for the next epoch given the current epoch's final
+// block height and epoch number. The view and timestamp for the simulated epoch are left empty since they can't
+// be easily simulated, so DO NOT USE CreatedAtBlockTimestampNanoSecs or InitialView from the returned EpochEntry.
+func (bav *UtxoView) simulateNextEpochEntry(currentEpochNumber uint64, currentEpochFinalBlockHeight uint64) (*EpochEntry, error) {
```

Comment:
Mention why we need this. I saw it but nice to have it there.

Reply:
Done.